### ISSUE TITLE
Pytorch, Ignite, Monai -> Monai Label Tutorial

### ISSUE THREAD

**Issue by danieltudosiu** [Project-MONAI/MONAILabel#1195]
**Is your feature request related to a problem? Please describe.**
Due to the complexities required by the creation of interfaces for the backend and frontend to interact the "information" flow within MONAI Label is heavily offuscated and hard to comprehend for development.

**Describe the solution you'd like**
Create the following four animations/figures:
- An UML-like figure of MONAI Label that is specific for the backend to complement the existing REST API one.
- Three transition animations that exemplify how you move from X to MONAI Label, where X is:
  - Pure Pytorch 
  - Ignite
  - MONAI

The last point is inspired by the [PyTorch Lighting animation](https://www.pytorchlightning.ai/#:~:text=optimizer.zero_grad()) which conveys a lot of information in a very succinct way.

**Additional context**
This came to be after I had explained MONAI Label to @MichelaA and a while ago Ignite to @diazandr3s. This would facilitate a faster and better understanding of MONAI Label and hopefully as a side-effect less issue.
